jms/src/test/resources/logback-test.xml

```diff
@@ -7,6 +7,8 @@
         </encoder>
     </appender>
 
+    <logger name="org.apache.activemq" level="info" />
```

Comment:
ActiveMQ with additional tests becomes very noisy at "debug" log level. The sheer volume of the output causes the tests to fail from hitting Travis output limits. Setting ActiveMQ logging to "info" instead.